etl/RetrosheetTableBuilder.py

```diff
@@ -105,43 +105,15 @@ def create_rts_tables(self):
 
         print("Create game_info table")
         self.cur.execute("""
-        	CREATE TYPE pitches_type AS ENUM ('pitches', 'count', 'none')
```

Comment:
As I mention above, the game info data looks like it's set up in a key-value format and without profiling the data I don't trust that what's going to show up there is going to be consistent. If we shove everything into a JSON blob it allows us to keep the data, without worrying to much about what it actually is.